code/drasil-example/projectile/lib/Drasil/Projectile/Body.hs

```diff
@@ -165,14 +174,67 @@ refDB = rdb citations concIns
 concIns :: [ConceptInstance]
 concIns = assumptions ++ funcReqs ++ goals ++ nonfuncReqs
 
--------------------------
+----------------------------------------
 -- Characteristics of Intended Reader --
--------------------------
+----------------------------------------
 
 charsOfReader :: [Sentence]
 charsOfReader = [phrase undergraduate +:+ S "level 1" +:+ phrase Doc.physics,
                  phrase undergraduate +:+ S "level 1" +:+ phrase calculus]
 
+-----------------
+--SystemContext--
+-----------------
+
+sysCtxIntro :: Contents
+sysCtxIntro = foldlSP
+  [refS sysCtxFig1, S "shows the" +:+. phrase sysCont,
+   S "A circle represents an entity external to the", phrase software
+   `sC` phraseNP (the user), S "in this case. A rectangle represents the",
+   phrase softwareSys, S "itself" +:+. sParen (short projectileTitle),
+   S "Arrows are used to show the data flow between the", phraseNP (system
+   `andIts` environment)]
+
+sysCtxDesc :: Contents
+sysCtxDesc = foldlSPCol [S "The interaction between the", phraseNP (product_
+   `andThe` user), S "is through an application programming" +:+.
+   phrase interface, S "The responsibilities of the", phraseNP (user 
+   `andThe` system), S "are as follows"]
+
+sysCtxUsrResp :: [Sentence]
+sysCtxUsrResp = map foldlSent [[S "Provide initial", pluralNP (condition `ofThePS`
+  physical), S "state of the", phrase motion, S "and the", plural inDatum, S "related to the",
```

Comment:
```suggestion
  physical), S "state of the", phrase motion `andThe` plural inDatum, S "related to the",
```
Some other `Sentence` combinators could be used here, but since the string here only contains text that is captured in a combinator, it should really be used!